src/FSharp.Analyzers.Cli/Program.fs

```diff
@@ -625,11 +628,16 @@ let main argv =
                         logger.LogError("Invalid `--project` argument. File does not exist: '{projPath}'", projPath)
                         exit 1
 
-                projects
-                |> List.map (fun projPath ->
-                    runProject client toolsPath properties projPath exclInclFiles severityMapping
-                )
-                |> Async.Sequential
+                async {
+                    let! loadedProjects = loadProjects toolsPath properties projects
+
+                    return!
+                        loadedProjects
+                        |> List.map (fun (projPath: FSharpProjectOptions) ->
+                            runProject client projPath exclInclFiles severityMapping
+                        )
+                        |> Async.Parallel
```

Comment:
Since we front load all projects we can use `Async.Parallel` to schedule as all the operations.